src/pages/docs/auth/identified-clients.mdx

```diff
@@ -9,6 +9,10 @@ When a client is assigned a trusted identity, that is, a `clientId`, then they a
 
 For example, assume you are building a chat application and want to allow clients to publish messages and be present on a channel. If each client is assigned a trusted identity by your server, such as a unique email address or UUID, then all other subscribed clients can trust any messages or presence events they receive in the channel as being from that client. No other clients are permitted to assume a `clientId` that they are not assigned in their Ably-compatible token. They are unable to masquerade as another `clientId`.
 
+<Aside data-type='see-evidence'>
+Message processing is designed to be transactional, therefore messages are either fully processed or not processed at all, preventing data corruption. [Click here for evidence 🔬](https://ably.com/docs/platform/architecture/fault-tolerance#core-layer)
```

Comment:
I'm not really sure what this is trying to say. Is it saying that a single message is processed atomically, or multiple messages are processed atomically. And I'm unsure how this claim relates to the surrounding text.

Reply:
I'll remove entirely (this one was a bot far fetched).